Userland/Utilities/disasm.cpp

```diff
@@ -16,7 +19,24 @@
 #include <LibMain/Main.h>
 #include <LibX86/Disassembler.h>
 #include <LibX86/ELFSymbolProvider.h>
-#include <string.h>
+
+struct Symbol {
+    size_t value;
```

Comment:
these should be default-init to 0.

Reply:
Should the value be FlatPtr?

Reply:
As far as I can tell, we mainly use FlatPtr in the Kernel, and `size_t` is large enough to hold any pointer as per C++ spec.